lib/cosmos/packets/packet_config.rb

```diff
@@ -279,6 +279,17 @@ def finish_packet()
         end
         @current_packet = nil
         @current_item = nil
+
+        if @limits_groups.length > 0
+          pkt = Packet.new("SYSTEM", "LIMITS_GROUPS", :BIG_ENDIAN, "Contains all the items defined by LIMITS_GROUPS")
+          @limits_groups.each do |group, items|
+            item = pkt.define_item(group, 0, 0, :DERIVED)
+            item.read_conversion = LimitsGroupConversion.new(items)
+            item.states = {"GREEN"=>0, "YELLOW"=>1, "RED"=>2}
```

Comment:
Obviously this is just a convention that the limits_group_conversion follows but it seems to work